Documentation/getting-started-guide.md

```diff
@@ -9,19 +9,32 @@ If you're not on Linux, you should do all of this inside [the rkt Vagrant](https
 package main
 
 import (
-	"log"
-	"net/http"
+        "github.com/coreos/go-systemd/daemon"
+        "log"
+        "net"
+        "net/http"
 )
 
 func main() {
 	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
 		log.Printf("request from %v\n", r.RemoteAddr)
 		w.Write([]byte("hello\n"))
 	})
-	log.Fatal(http.ListenAndServe(":5000", nil))
+	ln, err := net.Listen("tcp", ":5000")
+	if err != nil {
+	fmt.Fprintf(os.Stderr, "%v\n", err)
```

Comment:
Sorry did not see that

On Tue, Sep 13, 2016 at 10:19 AM, Sergiusz Urbaniak <
notifications@github.com> wrote:

> In Documentation/getting-started-guide.md
> <https://github.com/coreos/rkt/pull/2826#discussion_r78515264>:
>
> >  )
> >
> >  func main() {
> >  	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
> >  		log.Printf("request from %v\n", r.RemoteAddr)
> >  		w.Write([]byte("hello\n"))
> >  	})
> > -	log.Fatal(http.ListenAndServe(":5000", nil))
> > +	ln, err := net.Listen("tcp", ":5000")
> > +	if err != nil {
> > +	fmt.Fprintf(os.Stderr, "%v\n", err)
>
> The block inside the if statement needs indentation.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/coreos/rkt/pull/2826/files/f796cd6c9eec8135d6763786f4fbe87072c0ef97#r78515264>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGRc787UmT3nVsGImS8qkzTZG0QYvj8Zks5qplyBgaJpZM4I76GE>
> .
>



-- 

Alessandro Puccetti